src/app/data-planes/components/data-plane-traffic/ServiceTrafficCard.vue

```diff
@@ -21,90 +21,97 @@
-            :key="value"
+            <div
+              v-for="value in [props.traffic.http?.downstream_rq_1xx as (number | undefined) ?? 0].filter(item => item !== 0)"
+              :key="value"
+            >
+              <dt>{{ t('data-planes.components.service_traffic_card.1xx') }}</dt>
+              <dd>{{ t('common.formats.integer', { value: value }) }}</dd>
+            </div>
+            <div>
+              <dt>{{ t('data-planes.components.service_traffic_card.2xx') }}</dt>
+              <dd>{{ t('common.formats.integer', { value: props.traffic.http?.downstream_rq_2xx as (number | undefined) ?? 0 }) }}</dd>
+            </div>
+            <div
+              v-for="value in [props.traffic.http?.downstream_rq_3xx as (number | undefined) ?? 0].filter(item => item !== 0)"
+              :key="value"
+            >
+              <dt>{{ t('data-planes.components.service_traffic_card.3xx') }}</dt>
+              <dd>{{ t('common.formats.integer', { value: value }) }}</dd>
+            </div>
+            <div>
+              <dt>{{ t('data-planes.components.service_traffic_card.4xx') }}</dt>
+              <dd>{{ t('common.formats.integer', { value: props.traffic.http?.downstream_rq_4xx as (number | undefined) ?? 0 }) }}</dd>
+            </div>
+            <div>
+              <dt>{{ t('data-planes.components.service_traffic_card.5xx') }}</dt>
+              <dd>{{ t('common.formats.integer', { value: props.traffic.http?.downstream_rq_5xx as (number | undefined) ?? 0 }) }}</dd>
+            </div>
+          </template>
+          <template
+            v-else
           >
-            <dt>{{ t('data-planes.components.service_traffic_card.3xx') }}</dt>
-            <dd>{{ t('common.formats.integer', { value: value }) }}</dd>
-          </div>
-          <div>
-            <dt>{{ t('data-planes.components.service_traffic_card.4xx') }}</dt>
-            <dd>{{ t('common.formats.integer', { value: props.traffic.http?.downstream_rq_4xx as (number | undefined) ?? 0 }) }}</dd>
-          </div>
-          <div>
-            <dt>{{ t('data-planes.components.service_traffic_card.5xx') }}</dt>
-            <dd>{{ t('common.formats.integer', { value: props.traffic.http?.downstream_rq_5xx as (number | undefined) ?? 0 }) }}</dd>
-          </div>
-        </template>
-        <template
-          v-else
-        >
-          <div>
-            <dt>{{ t('data-planes.components.service_traffic_card.tx') }}</dt>
-            <dd>{{ t('common.formats.bytes', { value: props.traffic.tcp?.downstream_cx_rx_bytes_total as (number | undefined) ?? 0 }) }}</dd>
-          </div>
-          <div>
-            <dt>{{ t('data-planes.components.service_traffic_card.rx') }}</dt>
-            <dd>{{ t('common.formats.bytes', { value: props.traffic.tcp?.downstream_cx_tx_bytes_total as (number | undefined) ?? 0 }) }}</dd>
-          </div>
-        </template>
-      </dl>
```

Comment:
Just to note the majority of this red/green change was down to extra nesting, so there's far less change than it looks, the KSkeletonBox was also added here for a tiny loading state for the stats in the cards.